rust/kcl-lib/src/execution/kcl_value.rs

```diff
@@ -336,6 +311,7 @@ impl KclValue {
             KclValue::Number { .. } => "number",
             KclValue::String { .. } => "string (text)",
             KclValue::MixedArray { .. } => "array (list)",
+            KclValue::HomArray { .. } => "array (list)",
```

Comment:
should probably put the type of element in here too, now that we're tracking it

Reply:
It's returning a static str, so that's not easy to do, we can do it as a follow-up PR if you like (we can probably do better for lots of these strings)